python/cudaq/kernel/ast_bridge.py

```diff
@@ -2173,6 +2174,13 @@ def bodyBuilder(iterVal):
                 # Handle User-Custom Struct Constructor
                 cls, annotations = globalRegisteredTypes.getClassAttributes(
                     node.func.id)
+
+                for var, typ in annotations.items():
+                    origin = get_origin(typ)
+                    if origin is list or typ is list:
+                        self.emitFatalError(f'list type is not yet supported.',
```

Comment:
I think this message can be made clearer by wording -
```suggestion
                        self.emitFatalError(f'`list` type is not yet supported in data classes.',
```